components/Steps/Confirmation.jsx

```diff
@@ -10,8 +10,37 @@ const ConfirmationStep = ({
   successMessage,
 }) => {
   const router = useRouter();
-  const { ref } = router.query;
+  const { ref, personId } = router.query;
   if (!formSteps) return null;
+  if (personId)
+    return (
```

Comment:
`Confirmation` component should stay generic, with this we are adding business logic with a component that should stay as generic as possible.

We can do it in two ways:
- pass a confirmation component to the `formWizard` 
- define a confirmation step (as component) in the `data/create-new-person`

Also using `personId` as a check to show this might have problems with the `case` forms, as we are using the `personId` in the URL to define to which person we want to add the case.